src/components/Form.spec.js

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Form from './Form'
+
+describe('Form', () => {
+  const noop = jest.fn()
+  it('should render a group of radio buttons and a button', () => {
+    render(
+      <Form
+        questions={[
+          {
+            id: 1,
+            question: 'Dawn or dusk?',
+            options: ['Dawn', 'Dusk'],
+          },
+        ]}
+        onSaveHouse={noop}
+      />
+    )
+    expect(screen.getByLabelText('Dawn')).toBeInTheDocument()
+    expect(screen.getByLabelText('Dusk')).toBeInTheDocument()
+    expect(screen.getByRole('button', { name: /submit/i })).toBeInTheDocument()
+  })
+
+  it('should render a button with text "submit" when at last question and call onSaveHouse with form data', () => {
```

Comment:
I think these are 2 tests, rendering the next question and checking for correct rendering of the last question?

Reply:
True. Don't need those tests anyway, after I refactored the `Question` component.

Reply:
True. Will fix it.